index.bs

```diff
@@ -107,19 +113,69 @@ The {{GravitySensor}} class is an {{Accelerometer}}'s subclass. The {{GravitySen
 
 The <dfn>gravity</dfn> is a force that attracts an object to the center of the earth, or towards any other physical object having mass.
 
-Note: The <dfn>local coordinate system</dfn> of a mobile device is usually defined relative to
-the device's screen when the device in its default orientation (see figure below).
-
-<img src="images/accelerometer_coordinate_system.png" srcset="images/accelerometer_coordinate_system.svg" style="display: block;margin: auto;" alt="Accelerometer coordinate system.">
+Reference Frame {#reference-frame}
+----------------
+
+The [=local coordinate system=] for the [=spatial sensor=] classes defined
```

Comment:
Maybe

The reference frame for the sensor classes defined in this specification, is a local coordinate system, which can be defined as either THE device coordinate system, or THE screen coordinate system

Reply:
Reference frame also includes time, while 'local coordinate system' omits time component.

Reply:
can be rephrased: "The reference frame for the sensor classes defined in this specification,
**is expressed** in a [=local coordinate system=]"